content/themes/techno/assets/js/config.js

```diff
@@ -9,7 +9,7 @@ window.techno.config = {
 						title: 'Github'
 					},
 					{ 
-						href: 'https://twitter.com/iammenasco',
+						href: 'https://twitter.com/LtlFrmrProducts',
```

Comment:
This is exactly what we needed to change! Now it'll link them to the right twitter account.